Formula/rpm.rb

```diff
@@ -27,51 +13,40 @@ class Rpm < Formula
     sha256 "3a276b6a3f7273f8f88effbb057e8d65fc53fa9e812054951bfdd5ce618f54ac" => :yosemite
   end
 
-  depends_on "rpm2cpio" => :build
+  depends_on "beecrypt"
   depends_on "berkeley-db"
   depends_on "libmagic"
+  depends_on "libarchive"
   depends_on "popt"
-  depends_on "libtasn1"
+  depends_on "gnupg"
   depends_on "gettext"
+  depends_on "lua"
   depends_on "xz"
-  depends_on "ossp-uuid"
+  depends_on "zstd"
+  depends_on "pkg-config" => :run
 
   def install
     # only rpm should go into HOMEBREW_CELLAR, not rpms built
-    inreplace "macros/macros.in", "@prefix@", HOMEBREW_PREFIX
+    inreplace "macros.in", "@prefix@", HOMEBREW_PREFIX
+    inreplace "platform.in", "@prefix@", HOMEBREW_PREFIX
     args = %W[
       --prefix=#{prefix}
       --localstatedir=#{var}
-      --with-path-cfg=#{etc}/rpm
+      --sharedstatedir=#{var}/lib
+      --sysconfdir=#{etc}
       --with-path-magic=#{HOMEBREW_PREFIX}/share/misc/magic
-      --with-path-sources=#{var}/lib/rpmbuild
       --with-libiconv-prefix=/usr
-      --disable-openmp
-      --disable-nls
+      --enable-nls
       --disable-dependency-tracking
-      --with-db=external
-      --with-sqlite=external
-      --with-file=external
-      --with-popt=external
-      --with-beecrypt=internal
-      --with-libtasn1=external
-      --with-neon=internal
-      --with-uuid=external
-      --with-pcre=internal
-      --with-lua=internal
-      --with-syck=internal
+      --disable-plugins
+      --with-external-db
+      --with-crypto=beecrypt
       --without-apidocs
-      varprefix=#{var}
+      --with-vendor=homebrew
     ]
 
     system "./configure", *args
-    inreplace "Makefile", "--tag=CC", "--tag=CXX"
-    inreplace "Makefile", "--mode=link $(CCLD)", "--mode=link $(CXX)"
     system "make"
```

Comment:
Is the separate `make` step still needed or can we just run `make install`?

Reply:
We can probably just do a single make install.

Reply:
I'd try it without the `make` unless we know it's actually broken.